analytics/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "trustification-analytics"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+log = "0.4"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+tokio = { version = "1" }
+uuid = { version = "1", features = ["v4"] }
+
+clap = { version = "4", features = ["derive", "env"], optional = true }
+
+segment = { version = "0.2", optional = true }
+
+[features]
+default = ["analytics", "tracker"]
```

Comment:
Does this percolate as a default feature for random folks who might clone our repo? 

I think either we shouldn't, or ifff it tracks by default, it at least isn't wired to RHT's segment.io?

Reply:
Tracking is disabled by default, unless one provides the "write key", which is part of the deployment and not part of this code base.

So cloning and starting up will have tracking disabled.